languages/js/src/Host.ts

```diff
@@ -350,8 +359,14 @@ export class Host implements Required<DataFilteringQueryParams> {
     const instance = (await this.toJs(
       polarInstance
     )) as NullishOrHasConstructor;
-    const cls = this.getClass(name);
-    return instance instanceof cls || instance?.constructor === cls;
+
+    const userType = this.types.get(name);
+    if (userType !== undefined) {
```

Comment:
Is there a need to introduce this branch here? The old code assumed `getClass(name)` is never undefined -- the same assumption would hold for `types.get(name)`, no?

Reply:
Well, `getClass` throws the error if the class isn't registered, If we didn't fall through we'd have to handle the error here I guess. It wouldn't be valid to pass in a non-registered thing but you'd want the right error if you did.

Reply:
Makes sense. I suppose ideally `getClass(name)` would be replaced with `getUserType(name)`, but let's get this out.